Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Use the official Go image as the base image
+FROM golang:latest
```

Comment:
going on production? - don't forget to hard code the version which is known to work with your code.
:latest is good for development and tests.